scripts/build-docs.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e
+
+THIS_DIR="$( cd "$(dirname "$0")"; pwd -P )"
+cd "$THIS_DIR"/../help
+
+make clean && make html
```

Comment:
I think you plan to build the documentation also from *outside* the doc folder, right?

Reply:
Right, this just makes it possible to build the documentation without knowing about the Makefile or how to correctly use it. I would usually use `inotifywait` to run this script when file changes occur.